server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -166,7 +166,7 @@ private ConfigProperties() {
     public static final String PINSETTER_MAX_RETRIES = "pinsetter.retries.max";
     public static final int PINSETTER_MAX_RETRIES_DEFAULT = 10;
 
-    private static final String[] DEFAULT_TASK_LIST = new String[]{
+    public static final String[] DEFAULT_TASK_LIST = new String[]{
```

Comment:
At one point in this branch, I was using it in the test config to work around an issue in PinsetterKernel (which now has a comment in it explaining what I think is a better resolution). It eventually got reverted, but I didn't think opening up our default config was a problem so the scope never got changed back.